frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-builder/task-sidebar/Sidebar.scss

```diff
@@ -6,10 +6,10 @@ $overview-sidebar-width: 550px;
   box-shadow: var(--pf-global--BoxShadow--md);
   overflow-x: hidden;
   overflow-y: auto;
-  position: absolute;
-  right: 0;
+  position: fixed;
+  right: var(--pf-global--spacer--md);
   transition: opacity 175ms ease-out, transform 225ms ease-out;
-  top: 0;
+  top: 150px;
```

Comment:
We should avoid such magic numbers. In this case, this is the exact distance from the body to the sidebar. But this can depend on the font size, zoom factor, line breaks, the content.